File: cpp/if else/boilingwater.cpp
/*accept the temperature in degree celsius of water amd check whether it is boiling or not*/

#include <iostream>
using namespace std;

int main()
{
    float temp;
    cout << "Enter temperature:";
    cin >> temp;
    if (temp >= 100)
    {
        cout << "Water is boiled." << endl;
    }
    else
    {
        cout << "Water is yet to boil" << endl;
    }
    return 0;
}

File: java/if else/boilingwater.java
/*accept the temperature in degree celsius of water amd check whether it is boiling or not*/

import java.util.Scanner;

class BoilingWater {
    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the temperature in degree Celsius:");
        int temp = sc.nextInt();

        if (temp >= 100) {
            System.out.println("The water is boiling");
        } else {
            System.out.println("The water is not boiling");
        }
    }
}


